src/Cake.Issues.MsBuild.Tests/Cake.Issues.MsBuild.Tests.csproj

```diff
@@ -32,24 +33,28 @@
     <EmbeddedResource Include="Testfiles\XmlFileLoggerLogFileFormat\IssueWithLineZero.xml" />
     <EmbeddedResource Include="Testfiles\XmlFileLoggerLogFileFormat\IssueWithOnlyFileName.xml" />
     <EmbeddedResource Include="Testfiles\XmlFileLoggerLogFileFormat\IssueWithoutFile.xml" />
+    <EmbeddedResource Include="Testfiles\XmlFileLoggerLogFileFormat\MsBuildNulNul.log" />
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.8.0" />
-    <PackageReference Include="Cake.Core" Version="0.28.0" />
-    <PackageReference Include="Cake.Testing" Version="0.28.0" />
-    <PackageReference Include="Cake.Issues" Version="0.6.0" />
-    <PackageReference Include="Cake.Issues.Testing" Version="0.6.0" />
-    <PackageReference Include="Shouldly" Version="3.0.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
+    <PackageReference Include="Cake.Core" Version="0.30.0" />
```

Comment:
There's no need to update this dependency as it should link to the lowest version without breaking change for maximum compatibility. Same for other dependencies (the failing build is also caused by the Cake.Issues update)

Reply:
I agree.